configs/sst_identity_management-idm-client-tools.yaml

```diff
@@ -0,0 +1,20 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: IdM Client Tools
+  description: General software for enrolling system with external Identity Management server
+  maintainer: sst_identity_management
+
+  packages:
+  - freeipa-healthcheck
```

Comment:
I think freeipa-healthcheck would be better off in "FreeIPA Server - Domain Controller"

Reply:
Ok, I added it there too.